### PR TITLE
operators [CI] seldon-operator (1.13.1)

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
Update Seldon Operator to 1.13.1

Signed-off-by: Rafal Skolasinski <r.j.skolasinski@gmail.com>

### CODE REVIEW

File: operators/seldon-operator/1.13.1/manifests/seldon-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+  - email: hello@seldon.io
+    name: Seldon Technologies
+  maturity: stable
+  minKubeVersion: 1.16.0
+  provider:
+    name: Seldon Technologies
+  selector:
+    matchLabels:
+      name: seldon-operator
+  version: 1.13.1
```
Comment (mvalarh) [inline operators/seldon-operator/1.13.1/manifests/seldon-operator.clusterserviceversion.yaml:603]: Please add `replaces: seldon-operator.v1.12.0`

**Review by mvalarh — APPROVED** [review]

File: operators/seldon-operator/1.13.1/manifests/seldon-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (RafalSkolasinski) [reply operators/seldon-operator/1.13.1/manifests/seldon-operator.clusterserviceversion.yaml:604]: Done

**Review by cliveseldon — APPROVED** [review]

**Comment by RafalSkolasinski** [comment]
shall we merge? all tests seems to succeed

**Pull request merged**